shoulder_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_pan_joint
    joint_speed: 1
    motor:
        id: 0
        init: 512
        min: 0
        max: 900

shoulder_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_pitch_joint
    joint_speed: 1
    motor:
        id: 1
        init: 2048
        min: 926
        max: 3825

elbow_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow_roll_joint
    joint_speed: 1
    motor:
        id: 2
        init: 512
        min: 0
        max: 650

elbow_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow_pitch_joint
    joint_speed: 1
    motor:
        id: 3
        init: 512
        min: 200
        max: 890

wrist_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_roll_joint
    joint_speed: 1
    motor:
        id: 4
        init: 512
        min: 0
        max: 1023

wrist_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_pitch_joint
    joint_speed: 1
    motor:
        id: 5
        init: 512
        min: 200
        max: 900

gripper_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: gripper_roll_joint
    joint_speed: 1
    motor:
        id: 6
        init: 512
        min: 0
        max: 1023

gripper_grip_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: gripper_grip_joint
    joint_speed: 1
    motor:
        id: 7
        init: 424
        min: 240
        max: 424

